samplePDF/samplePDFFDBase.cpp

```diff
@@ -207,120 +332,55 @@ bool samplePDFFDBase::IsEventSelected(const std::vector< std::string >& Paramete
 bool samplePDFFDBase::IsEventSelected(const std::vector< std::string >& ParameterStr,
                                       const std::vector< std::vector<double> > &SelectionCuts,
                                       const int iSample, const int iEvent) {
-// ************************************************
-
+  // ************************************************
+  
   double Val;
-
   for (unsigned int iSelection=0;iSelection<ParameterStr.size();iSelection++) {
     
     Val = ReturnKinematicParameter(ParameterStr[iSelection], iSample, iEvent);
-    //DB If multiple return values, it will consider each value seperately
-    //DB Already checked that SelectionCuts vector is correctly sized
-    
-    //DB In the case where SelectionCuts[0].size()==3, Only Events with Val >= SelectionCuts[iSelection][1] and Val < SelectionCuts[iSelection][2] are considered Passed
-	//ETA - also check whether we're actually applying a lower or upper cut by checking they aren't -999
-	if(Val >= SelectionCuts[iSelection][1] && SelectionCuts[iSelection][0] != -999){
-	  //std::cout << "Cutting event as " << Val << " is greater than " << SelectionCuts[iSelection][1]
-	  return false;
-	}
-	else if(Val < SelectionCuts[iSelection][0] && SelectionCuts[iSelection][1] != -999){
-	  return false;
-	}
+    if(Val >= SelectionCuts[iSelection][1] && SelectionCuts[iSelection][0] != -999){
```

Comment:
why -999?

I think there are macros (yuck) of 'BAD_FLOAT' etc, defined, so I assume its that, but then you should check equality with the macro value not a hard coded value that happens to be the same as the macro at some point.

-999 is also not a great bad value, not the worst, but not great.